lib/shopify-cli/commands/generate.rb

```diff
@@ -23,8 +23,9 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:page}}: Generate a new page in your app with the specified page name.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}}
-
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}} or
```

Comment:
Just thinking ahead about the scalability of this model, where under "Usage" we list possible combinations of commands, separated by "or" at the end of the line. What happens if we add more options? Just keep adding lines? Or is there a more extensible content model?

To be clear, I think this is **totally good to go today**, but could we make an issue to investigate how to grow this model sustainably in the longer term?

Reply:
Agree, I told Katie I'd look into a longer term/scalable design for this type of help content.